cilium-cli/connectivity/check/features.go

```diff
@@ -160,9 +160,6 @@ func (ct *ConnectivityTest) extractFeaturesFromCiliumStatus(ctx context.Context,
 			if f.HostPort != nil {
 				result[features.KPRHostPort] = features.Status{Enabled: f.HostPort.Enabled}
 			}
-			if f.GracefulTermination != nil {
-				result[features.KPRGracefulTermination] = features.Status{Enabled: f.GracefulTermination.Enabled}
-			}
```

Comment:
It looks like this feature is never inspected by the connectivity tests, so no concerns in dropping it also when running against older versions of Cilium :+1: